_posts/2017-09-18-impl-future-for-rust.md

```diff
@@ -0,0 +1,312 @@
+---
+layout: post
+title: "impl Future for Rust"
+author: Aaron Turon
+description: "The Rust community is going to finish out its 2017 roadmap with a bang—and we want your help!"
+---
+
+The Rust community has been hard at work on our [2017 roadmap], but as we come
+up on the final quarter of the year, we're going to kick it into high gear—and
+we want you to join us!
+
+[2017 roadmap]: https://github.com/rust-lang/rfcs/pull/1774
+
+Our goals for the year are ambitious:
+
+* [Rust should have a lower learning curve](https://github.com/rust-lang/rust-roadmap/issues/3).
+* [Rust should have a pleasant edit-compile-debug cycle](https://github.com/rust-lang/rust-roadmap/issues/1).
+* [Rust should provide a solid, but basic IDE experience](https://github.com/rust-lang/rust-roadmap/issues/2).
+* [Rust should provide easy access to high quality crates](https://github.com/rust-lang/rust-roadmap/issues/9).
+* [Rust should be well-equipped for writing robust, high-scale servers](https://github.com/rust-lang/rust-roadmap/issues/10).
+* [Rust should have 1.0-level crates for essential tasks](https://github.com/rust-lang/rust-roadmap/issues/11).
+* [Rust should integrate easily into large build systems](https://github.com/rust-lang/rust-roadmap/issues/12).
+* [Rust's community should provide mentoring at all levels](https://github.com/rust-lang/rust-roadmap/issues/13)
+
+**To finish off these goals, we intend to spend the rest of the year focused
+purely on "implementation" work—which doesn't just mean code!** In particular, we
+are effectively spinning down the [RFC process] for 2017, after having merged
+[almost 90] RFCs this year!
+
+[RFC process]: https://github.com/rust-lang/rfcs#rust-rfcs
+[almost 90]: https://github.com/rust-lang/rfcs/pulls?utf8=%E2%9C%93&q=is%3Apr%20merged%3A%3E2017-01-01
+
+So here's the plan. Each Rust team has put together several *working groups*
+focused on a specific sub-area. Each WG has a leader who is responsible for
+carving out and coordinating work, and a dedicated chat channel for getting
+involved. We are working hard to divvy up work items into many shapes and sizes,
+and to couple them with mentoring instructions and hands-on mentors. **So if
+you've always wanted to contribute to Rust but weren't sure how, this is the
+perfect opportunity for you.** Don't be shy—we want and need your help, and, as
+per our roadmap, our aim is mentoring at *all* levels of experience.
+
+## A few points of order
+
+There are a few online venues for keeping in the loop with working group activity:
+
+- There is a [dedicated Gitter community](https://gitter.im/rust-impl-period/)
+  with channels for each working group, as well as
+  a [global channel](https://gitter.im/rust-impl-period/Lobby) for talking about
+  the process as a whole, or getting help finding your way to a working group.
+
+- The brand-new [findwork](https://www.rustaceans.org/findwork) site, which
```

Comment:
Could use https://www.rustaceans.org/findwork/impl to go straight to the impl period issues